components/modal/README.md

```diff
@@ -0,0 +1,170 @@
+Modal
+=======
+
+The modal is used to create an accessible modal over an application.
+
+**Note:** The API for this modal has been mimicked to resemble `react-modal`.
+
+## Usage
+
+Render a screen overlay with a modal on top.
+```jsx
+	<Modal
+		title="My Modal"
+		onRequestClose={ closeFunction }
+		isOpen={ openState }
+		>
+		<ModalContent>
+			<p id="modal-description">This modal is meant to be awesome!</p>
```

Comment:
I've added the `aria.describedby` to the example to demonstrate how this prop can be used to improve accessibility.

Reply:
We still should never encourage / allow static `id`. We should update the example to use `withInstanceId` to guarantee a unique ID if we want the ID to be included. 

https://github.com/WordPress/gutenberg/tree/master/components/higher-order/with-instance-id

Alternatively, since there's decent overhead in doing so, maybe this is something we should bake as behavior on behalf of the developer?